### PR TITLE
Updated the ReadMe with repo_path information from PR #7

### PR DESCRIPTION
**Pull request by thespain**
Please see the updated changes in the ReadMe for information on repo_path from PR #7.

### CODE REVIEW

**Comment by thespain** [comment]
I moved the example into its own block. What do you think?

File: README.md [hunk 1]
```diff
@@ -237,6 +237,7 @@ byproduct of how Sinatra / Rack do their logging.
 http://localhost:4567/deploy
 tree: tree-planter
 repo_url: https://github.com/genebean/tree-planter.git
+repo_path: tree-planter
```
Comment (genebean) [inline README.md]: Remove this line please

File: README.md [hunk 2]
```diff
@@ -247,6 +248,7 @@ Cloning into 'tree-planter'...
...
```
Comment (genebean) [inline README.md]: Remove this line please

**Comment by thespain** [comment]
I can certainly remove those if you like, but I left them in there because I double checked the current master branch and as it is now when those two curl commands are run, then you do indeed get the `repo_path` output. Unless _I did something dumb_, then please enlighten me.  Whatever you want to do is fine though, just let me know! 😄

**Comment by genebean** [comment]
You are totally right, thanks for fixing that!

**Pull request merged**